spec/controllers/recipe_revisions_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe RecipeRevisionsController do
+  let!(:user) { recipe.user }
```

Comment:
well i guess thats really a personal thing here. for me its fine (even in spec files >500 lines).